packages/device_info_plus/device_info_plus/lib/src/device_info_plus_web.dart

```diff
@@ -47,7 +48,6 @@ class DeviceInfoPlusWebPlugin extends DeviceInfoPlatform {
 
 /// Property is missing.
 /// Ticket: https://github.com/dart-lang/web/issues/192
-/// Probably won't be an int? in the future!
 extension on html.Navigator {
-  external int? get deviceMemory;
+  external double? get deviceMemory;
```

Comment:
Looking at the linked ticket, it is possible that `deviceMemory` is now part of the `web` package navigator, but haven't looked into that yet.

Reply:
I saw the linked ticket but I decided to focus on the webasm compilation. Do you need me to add something regarding this?

Reply:
Yes, I think this extension can be replaced by web's `deviceMemory`: https://pub.dev/documentation/web/latest/web/Navigator/deviceMemory.html